upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template

```diff
@@ -289,14 +289,14 @@ spec:
           command:
             - ./cluster-autoscaler
             - --balance-similar-node-groups={{ .BalanceSimilarNodeGroups }}
-            - --cloud-provider={{ $.CloudProvider }}
-            {{ if (eq $.CloudProvider "aws") }}
+            - --cloud-provider={{ GetCloudProvider }}
```

Comment:
Could we name this CloudProvider (for the templates)?

Reply:
I patterned this off of GetNodeInstanceGroups and HasHighlyAvailableControlPlane. Should template functions that call getters be named as getters or attributes?

Reply:
A good question - I was more just thinking it was odd to go from `$.CloudProvider` to `GetCloudProvider`.  If there's precedent, what you've done makes sense